### DISCUSSION TITLE
why can't I get past lock screen upon restart local or remotely?

### DISCUSSION THREAD

**Discussion by anotherITbro** [rustdesk/rustdesk#3846] [Q&A]
Recently installed:
- RustDesk server
- RustDesk relay
-2 Windows 10 client (fully installed)
-1 Windows 11 client(fully installed)

If the machines are already logged in, connects perfectly fine.

But once restarted, and lock screen is clicked no login for password or pin appears just the lock screen.(I have to physically be at the keyboard of that computer)
I made sure to:
Allow traffic from the specified ports on the RustDesk tutorial via Kaspersky Total Security's firewall so definitely connects

**Comment by rustdesk** [comment]
send `ctrl+alt+del` ?

![image](https://user-images.githubusercontent.com/71636191/228714110-5106dcff-5f56-4eaa-bba7-28e27f8dcd0d.png)

**Reply by anotherITbro** [reply]
![20230329_224549.jpg](https://user-images.githubusercontent.com/129349840/228715256-30437c56-bfea-472a-bc8c-dee2ef077c28.jpg)

> Hmmm I'm not seeing that...
I believe I did install the latest

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/1244

**Reply by anotherITbro** [reply]
I look through the threads this linked to. The software is installed on

Client is installed on both Local device and remote PC I'm connecting to

**Reply by anotherITbro** [reply]
It seems very unclear what The solution is to the problem. I have not used the portable version.
